Makefile.darwin

```diff
@@ -19,9 +19,8 @@ endif
 
 FINCH_IMAGE_LOCATION := $(OS_OUTDIR)/$(FINCH_OS_BASENAME)
 FINCH_IMAGE_DIGEST := "sha512:$(FINCH_OS_DIGEST)"
-FEDORA_YAML := fedora.yaml
 
-dependencies: download.os install.lima-dependencies lima-socket-vmnet lima-template
+dependencies: download.os install.lima-dependencies lima-socket-vmnet
```

Comment:
`lima-template` isn't really a platform core dependency. It is used for generating a `fedora.yaml` configuration to use for `finch-core` end-to-end testing.